code/game/machinery/mass_driver.dm

```diff
@@ -71,3 +72,6 @@
 	if(stat & (BROKEN|NOPOWER))
 		return
 	drive()
+
+// More procs at /singulostation/code/game/machinery/mass_driver.dm
+// ~ MaltVinegar
```

Comment:
Not an issue, but do we want to add this kind of comment to every case where an existing type has new things added by us?

Might want to remove the signature though, maybe phrase it in a more generic way rather than "More procs", like "See also", since this can also include vars and subtypes.